packages/renderer/src/AppNavigation.svelte

```diff
@@ -2,11 +2,12 @@
 
 <script lang="ts">
 import { Tooltip } from '@podman-desktop/ui-svelte';
```

Comment:
issue: I see no new test in AppNavigation.spec.ts for checking the new behaviour